NarutoLutaDeNinjas-Map/src/main/java/br/tec/db/NinjaDeNinjutsu.java

```diff
@@ -0,0 +1,36 @@
+package br.tec.db;
+
+public class NinjaDeNinjutsu extends Personagem implements Ninja {
+
+    public NinjaDeNinjutsu(String nome, int vida) {
+        super(nome, vida);
+    }
+
+    @Override
+    public void usarJutsu(String jutsuKeyMap, Personagem personagemAtacado) {
+        if (this.getJutsus().containsKey(jutsuKeyMap)) {
+
+            Jutsu jutsu = getJutsus().get(jutsuKeyMap);
+
+            if(verificaSeTemChakra(jutsu)) {
```

Comment:
Não é obrigatório, mas na orientação a objetos do java temos as keywords "this" e "super" que indica o nível que estao sendo  acessandos. "this" é usado para metodos e campos referentes ao mesmo nivel da classe, "super" é usado para indicar que é da classe "pai". Eu gosto de colocar para facilitar a leitura, mas não é obrigatorio.